### PR TITLE
Bug/fix bugs found

### PR DESCRIPTION
**Pull request by xkevin190**

### CODE REVIEW

**Review by luisan00 — APPROVED** [review]

File: android/gradle.properties [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+FLIPPER_VERSION=0.33.1
+
+
+org.gradle.daemon=true
+org.gradle.configureondemand=true
+org.gradle.jvmargs=-Xmx4g -XX:MaxPermSize=2048m -XX:+HeapDumpOnOutOfMemoryError -Dfile.encoding=UTF-8
+MYAPP_RELEASE_STORE_FILE=my-upload-key.keystore
+MYAPP_RELEASE_KEY_ALIAS=my-key-alias
+MYAPP_RELEASE_STORE_PASSWORD=hola 1234
+MYAPP_RELEASE_KEY_PASSWORD=hola 1234
```
Comment (jeandudey) [inline android/gradle.properties:37]: :eyes:

Comment (luisan00) [reply android/gradle.properties:37]: :dancing_men:

File: src/views/LoadWallet/RestoreWithPin.js [hunk 2]
```diff
@@ -36,6 +36,7 @@ class RestoreWithPin extends Component {
           callback();
           return;
         }
+        callback();
```
Comment (luisan00) [inline src/views/LoadWallet/RestoreWithPin.js:39]: test it

File: src/database/index.js [hunk 3]
```diff
@@ -107,6 +107,7 @@ export default class Database extends CoreDatabase {
       const userData = await this.getUserData();
       resolve({ user: userData, seed: this.seed.objects('Seed') });
     } catch (err) {
+      console.log("err123", err);
```
Comment (luisan00) [inline src/database/index.js:110]: test it

File: src/views/home/index.js [hunk 4]
```diff
@@ -201,10 +203,10 @@ class index extends Component {
             );
             const infoData = this.getContactInformation(chat);
             const messages = Object.values(chat.messages);
-            const lastmessage = this.getDataTypeMessage(messages[messages.length - 1]);
 
-            const lasTime = Number(messages[messages.length - 1].timestamp);
             if (messages.length !== 0) {
+              const lastmessage = this.getDataTypeMessage(messages[messages.length - 1]);
+              const lasTime = Number(messages[messages.length - 1].timestamp);
```
Comment (luisan00) [inline src/views/home/index.js:209]: test it

**Review by jeandudey — APPROVED** [review]

**Pull request merged**